drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -186,7 +183,7 @@ def studio_view(self, context):
 
         js_templates = loader.load_unicode('/templates/html/js_templates.html')
         help_texts = {
-            field_name: self._(field.help)
+            field_name: self.ugettext(field.help)
```

Comment:
@asadiqbal08 

> For just testing, you can mark any string(s) in student_view with self.ugettext. You will get the translated string(s)

Yes, I did that yesterday and it worked fine :)

Thanks for the heads-up about translation support for `studio_view` being missing. I think the only thing that's left to address now is [this comment](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/66#discussion_r58084737).